net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+         hcs_install iem_ambi_install iem_bin_ambi_install iemlib_install \
+         iemguts_install iem_adaptfilt_install iemmatrix_install iemxmlrpc_install \
+         iem_delay_install iem_roomsim_install iem_spec2_install iem_tab_install \
+         iemgui_install jasch_lib_install loaders-libdir_install lyonpotpourri_install \
+         mapping_install markex_install maxlib_install mjlib_install \
+         moocow_install moonlib_install motex_install mrpeach_install oscx_install \
+         pan_install pdcontainer_install pddp_install pdlua_install pdogg_install \
+         plugin_install pmpd_install rjlib_install sigpack_install smlib_install \
+         tof_install unauthorized_install vbap_install windowing_install zexy_install
+
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231012'
+    commit: 27b66ed5efd4de7cb0fd7ad33cf46c60cb8201fc
+
+- name: pd-abstractions
+  buildsystem: simple
+  subdir: abstractions
+  build-commands:
+  - make -j ${FLATPAK_BUILDER_N_JOBS} DESTDIR=/ prefix=/app install
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231012'
+    commit: 27b66ed5efd4de7cb0fd7ad33cf46c60cb8201fc
+
+- name: pd-l2ork-integration
+  buildsystem: simple
+  build-commands:
+  # These commands run in the 'subdir' that we set, but the sources appear at
+  # the top level hence the `../..` everywhere.
+  - mkdir -p /app/lib/pd-l2ork/bin/nw
+  - tar -x -f nwjs.tar.gz -C /app/lib/pd-l2ork/bin/nw --strip-components=1
+  # wrapper script
+  - install -m 755 pd-wrapper.sh /app/bin
+  # .desktop file (so we appear in list of applications)
+  - install -d /app/share/applications/
+  - install -m 644 net.pdl2ork.PdL2Ork.desktop /app/share/applications/
+  # AppStream metadata (for app store / package manager)
+  - install -d /app/share/metainfo/
+  - install -m 644 net.pdl2ork.PdL2Ork.metainfo.xml /app/share/metainfo
+  # icon (for everyone)
+  - install -d /app/share/icons/hicolor/scalable/apps
+  - install -m 644 net.pdl2ork.PdL2Ork.svg /app/share/icons/hicolor/scalable/apps/
+  # add mime type to .pd files
+  - install -d /app/share/mime/packages/
+  - install -m 644 packages/linux_make/pd-l2ork.xml /app/share/mime/packages/net.pdl2ork.PdL2Ork.xml
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231012'
+    commit: 27b66ed5efd4de7cb0fd7ad33cf46c60cb8201fc
+  - type: file
+    # x86_64 for now only. Still waiting for a comparable arm7 version for RPi.
+    url: https://dl.nwjs.io/v0.67.1/nwjs-v0.67.1-linux-x64.tar.gz
+    sha256: 98a24a98ca7252ad9d0b56a9fc9518e76f128b436b1555355b808907f7b66bd4
+    dest-filename: nwjs.tar.gz
+  - type: file
+    path: ./pd-wrapper.sh
+  - type: file
+    path: net.pdl2ork.PdL2Ork.desktop
+  - type: file
+    path: net.pdl2ork.PdL2Ork.metainfo.xml
+  - type: file
+    path: net.pdl2ork.PdL2Ork.svg
+  - type: file
+    path: default.settings
```

Comment:
since these files are in the repo upstream, why having theme here?

Reply:
I am still trying to understand when you issue build/install commands to a module without a subfolder: option, what folder am I in? In the root of the cloned git folder where thing was checked out, or in the folder from which I ran the script? Some of the commands suggest former, and some latter. Yet all of them appear to work.